src/server/migrations/20220123184131_create_products_table.js

```diff
@@ -0,0 +1,36 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('products', function (table) {
+    table.increments();
```

Comment:
Does this serve as Product ID?

Reply:
Yes, that is supposed to be the MYSQL auto increment.

Reply:
table.string('name').notNullable().unique();  <- Added unique to name.